hack/imager.sh

```diff
@@ -17,3 +17,16 @@ prepare_extension_images() {
 
     echo "${extensions_dir}"
 }
+
+# returns the value to use for the raw disk image generated by imager.
+# we use the smallest possible value for metal and sbc, a 10GB value for all cloud images.
+determine_image_disk_size() {
```

Comment:
I wonder if we should put the default into Go code, this way you have the ability still to run `imager` with `docker run` without specifying the size? 

this might be easier to support and find, as these will be constants in the code. 

iirc we have anyways big `switch` statement for each platform in the code, we could put the size there

(we can keep the arg as well to override the default)

Reply:
We create the raw disk before the switch statement, but I do think there's a better way to do this. Gimme just a few.